src/Command/Ingester/CreateUserCommand.php

```diff
@@ -0,0 +1,461 @@
+                }
+            }
+
+            if (!empty($userAggregratedCsvDt)) {
+                $this->writeCsvData($path = $automateCsvPath."/users_aggregated.csv", $userCsvHead, $userAggregratedCsvDt);
+            }
+            if (!empty($assgAggregratedCsvDt)) {
+                $this->writeCsvData($path = $automateCsvPath."/assignments_aggregated.csv", $assignmentCsvHead, $assgAggregratedCsvDt);
+            }
+
+            
+            if (!$userDtoCollection->isEmpty()) {
+                $this->userRepository->insertMultiple($userDtoCollection);
+            }
+            if (!$assignmentDtoCollection->isEmpty()) {
+                $this->assignmentIngester->ingest($assignmentDtoCollection);
+            }
+            
+            $this->symfonyStyle->success(
+                sprintf(
+                    "'%s' Users have been successfully added.", $noOfUsersCreated
+                )
+            );
+
+        } catch (Throwable $exception) {
+            $this->symfonyStyle->error($exception->getMessage());
+
+            return 1;
+        }
+
+        return 0;
+    }
+
+    private function initializeLineItemIdsOption(InputInterface $input): void
+    {
+        $lineItemIds = array_filter(
+            explode(',', (string)$input->getOption(self::OPTION_LINE_ITEM_IDS)),
+            static function (string $value): bool {
+                return !empty($value) && (int)$value > 0;
+            }
+        );
+        if (empty($lineItemIds)) {
+            throw new InvalidArgumentException(
+                sprintf("Invalid '%s' option received.", self::OPTION_LINE_ITEM_IDS)
+            );
+        }
+
+        $this->lineItemIds = array_map('intval', $lineItemIds);
+    }
+
+    private function initializeLineItemSlugsOption(string $lineItemSlugs): void
+    {
+        if (!empty($lineItemSlugs)) {
+            $this->lineItemSlugs = array_filter(
+                explode(',', (string)$lineItemSlugs),
+                static function (string $value): bool {
+                    return !empty($value);
+                }
+            );
+        }
+        if (empty($this->lineItemSlugs)) {
+            throw new InvalidArgumentException(
+                sprintf("Invalid '%s' option received.", self::OPTION_LINE_ITEM_SLUGS)
+            );
+        }
+    }
+
+    private function initializeUserPrefixOption(InputInterface $input): void
+    {
+        $this->userPrefix = array_filter(
+            explode(',', (string)$input->getArgument('user-prefix')),
+            static function (string $value): bool {
+                return !empty($value);
+            }
+        );
+
+        if (empty($this->userPrefix)) {
+            throw new InvalidArgumentException(
+                sprintf("User Prefix is a required argument")
+            );
+        }
+    }
+
+    private function createUserPassword(): string
+    {
+        return substr(str_shuffle("0123456789abcdefghijklmnopqrstvwxyz"), 0, 8);
```

Comment:
if we need a secure password it should be refactored

Reply:
This don't need to be super secure since these users are only for QA. 
Maybe generate hashes and take part of then would work as well. I would also move this this password generator to a separate component.